Formula/ktor.rb

```diff
@@ -0,0 +1,26 @@
+class Ktor < Formula
+  desc "CLI tool that creates porjects with ktor framework setup"
+  homepage "https://github.com/ktorio/ktor-cli"
+  url "https://github.com/ktorio/ktor-cli/archive/refs/tags/0.0.1-tmp.tar.gz"
+  sha256 "bdb311910a3afcebe3428aee6fd1ef8c19ac670d089f362a6fb89030c3edf873"
+  license "Apache-2.0"
+
+  depends_on "gradle" => :build
+
+  uses_from_macos "curl"
+  uses_from_macos "unzip"
+
+  def install
+    if OS.mac?
+      system "gradle", "linkReleaseExecutableMacosX64"
+      bin.install "build/bin/macosX64/releaseExecutable/ktor-cli-generator.kexe" => "ktor"
+    else
+      system "gradle", "linkReleaseExecutableLinuxX64"
+      bin.install "build/bin/linuxX64/releaseExecutable/ktor-cli-generator.kexe" => "ktor"
+    end
+  end
+
+  test do
+    system "ktor", "generate", "projectName"
```

Comment:
Can we do some assertions here about files?

Reply:
Will do, thanks!